### PR TITLE
docs(cn): translate `Global navigation failures` section of `Waiting for the result of a Navigation` Chapter into Chinese

### PR DESCRIPTION
**Pull request by yionr**

### CODE REVIEW

File: packages/docs/.vitepress/translation-status.json
Comment (Jinjiang) [inline packages/docs/.vitepress/translation-status.json:1]: Same as https://github.com/vuejs/router/pull/2090#discussion_r1436745733

File: packages/docs/zh/guide/advanced/navigation-failures.md [hunk 1]
```diff
@@ -64,9 +64,9 @@ if (isNavigationFailure(failure, NavigationFailureType.aborted)) {
 
 <!-- TODO: translation -->
 
-## Global navigation failures
+## 全局导航故障
 
-You can detect global navigation failures globally by using the [`router.afterEach()` navigation guard](./navigation-guards.md#Global-After-Hooks):
+你可以用 [`router.afterEach()` 导航指南](./navigation-guards.md#Global-After-Hooks) 检测全局导航故障：
```
Comment (Jinjiang) [inline packages/docs/zh/guide/advanced/navigation-failures.md:69]:
```suggestion
你可以用 [`router.afterEach()` 导航守卫](./navigation-guards.md#Global-After-Hooks)检测全局导航故障：
```

**Review by Jinjiang — CHANGES_REQUESTED** [review]

**Comment by Jinjiang** [comment]
@yionr well done. I left some comments in all your 4 PRs. Thanks.

**Comment by yionr** [comment]
> @yionr well done. I left some comments in all your 4 PRs. Thanks.

You are welcome. Thanks you all guys the hark work that I can get started of this framework such fast

**Review by Jinjiang — APPROVED** [review]

**Pull request merged**